Java/README.md

```diff
@@ -36,4 +36,4 @@
 |           Prim's Algorithm             |       [link](./Graphs/PrimsAlgorithm.java)                |
```

Comment:
This file also needs to be removed from this PR

Reply:
@kaustubhgupta This file Prim's Algorithm is added as a part of the PR https://github.com/kaustubhgupta/Hacktoberfest2020_/pull/364/files. Do you still want me to remove it?